openapi/paths/credit-memos@{id}.yaml

```
@@ -69,3 +69,35 @@ put:
       $ref: ../components/responses/NotFound.yaml
     '422':
       $ref: ../components/responses/ValidationError.yaml
+patch:
+  x-products:
+    - Core
+  tags:
+    - Credit memos
+  summary: Update credit memo, allocation, or both
+  operationId: PatchCreditMemo
+  x-sdk-operation-name: patch
+  description: |-
+    Updates a credit memo, credit memo allocation, or both credit memo and allocation based a specified ID.
```

sorry, I made a typo here
```suggestion
    Updates a credit memo, credit memo allocation, or both credit memo and allocation with a specified ID.
```